### DISCUSSION TITLE
Please tell me your favorite Music Artists!! 貴方の好きなアーティストを教えてください！

### DISCUSSION THREAD

**Discussion by ChanYu1224** [NAIST-SE/AttractingContributors#408] [General]
Please tell me your favorite music artists!
All genre are welcome!

-----
貴方の好きなアーティストを教えてください！
どのようなジャンルでも大歓迎です。

----
My fork is here:
https://github.com/ChanYu1224/AttractingContributors-MusicArtists

**Comment by AtropsCooper** [comment]
Mine is Nhato.
Help me please if you'd like to 🥺
https://github.com/AtropsCooper/AttractingContributors-Random

**Comment by SInadumi** [comment]
Hi, writing my fav artists.
Please tell me your fav one.
https://github.com/SInadumi/AttractingContributors-Musician

**Comment by Kotjik** [comment]
Hey, I contributed to your repo - would you mind adding something to mine as well? 🙂
Here's my repo: https://github.com/Kotjik/AttractingContributors-FavoriteCharacter
